frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -226,25 +228,41 @@ private void refreshModel(
 
     setModel(new GraphTableModel(repositoryGraph));
 
-    Set<String> skippedBranchNames = repositorySnapshot.getSkippedBranchNames();
-    if (skippedBranchNames.nonEmpty()) {
-      val notification = getSkippedBranchesNotification(repositorySnapshot, gitRepository);
-      VcsNotifier.getInstance(project).notify(notification);
-    }
+    if (!macheteFileIsOpened(macheteFilePath)) {
```

Comment:
I am not convinced of this idea - to show or not to show a notification depending on some file being opened. For me, it looks like a bug.

Reply:
Hmmm the problem is that generally we want to notify the user of duplicate and/or non-existent branches... but it's rather annoying when the notifications pop up during an edit ☹️ compare it to errors during editing code (no notification, just red squiggles) vs during build from IntelliJ (probably some notification-like stuff shows up)

Reply:
@mkondratek but that is the idea of this issue:
https://github.com/VirtusLab/git-machete-intellij-plugin/issues/1238
> Notifications about duplicate/non-existent branch should NOT be displayed when .git/machete is opened in editor

Reply:
☝🏻 Yeah, hence I'm taking the burden of defending the idea 😅